### PR TITLE
add probe functionality in AMD/Renoir platform.

### PR DESCRIPTION
**Pull request by DINESHKUMARK1**
Add probe functionality in AMD/Renoir platform.

### CODE REVIEW

**Comment by sofci** [comment]
Can one of the admins verify this patch?
> reply **test&nbsp;this&nbsp;please** to run this test once

**Comment by DINESHKUMARK1** [comment]
test this please

File: src/drivers/amd/renoir/acp_dma.c [hunk 1]
```diff
@@ -201,8 +201,11 @@ int dma_setup(struct dma_chan_data *channel,
 			} else {
 				/* Capture */
 				dma_cfg->base = dma_config_dscr[dscr_strt_idx].src_addr;
```
Comment (dbaluta) [inline src/drivers/amd/renoir/acp_dma.c:203]: I think you should split this into 2 patches. 

First one modifying acp_dma.c with a proper explanation of why the change is needed. Second patch actually enabling CONFIG_PROBE and friends.

**Review by sunilkumardommati — APPROVED** [review]

**Review by bhiregoudar — APPROVED** [review]

**Comment by lgirdwood** [comment]
test this please

**Review by lgirdwood — APPROVED** [review]
All of @dbaluta comments now addressed.
